connect-processors/cloud/streams/cloud-events.yaml

```
@@ -0,0 +1,201 @@
+        let topic = $category.or($acapy_topic)
+
+        let cloudapi_topic = $topic_mapping.get($topic).or("undefined")
+        root.cloudapi_topic = $cloudapi_topic
+
+        # Topic-specific transformations
+        if $cloudapi_topic == "connections" {
+            root.payload.state = $payload.rfc23_state.or($payload.state) # "deleted" only occurs in state, not rfc23_state
+        }
+        if $cloudapi_topic == "endorsements" {
+            root.payload.state = $payload.state.replace_all("_", "-")
+        }
+        if $category == "issue_credential_v2_0" {
+            root.payload.credential_exchange_id = "v2-" + $payload.cred_ex_id
+            root.payload.credential_id = root.payload.credential_exchange_id
+            root.payload.state = $payload.state.replace_all("_", "-")
+
+            let indy = if $payload.exists("by_format.cred_offer.indy") {
+                $payload.by_format.cred_offer.indy
+            } else if $payload.by_format.exists("by_format.cred_proposal.indy") {
+                $payload.by_format.cred_proposal.indy
+            } else {
+                {}
+            }
+            root.payload.schema_id = $indy.schema_id
+            root.payload.credential_definition_id = $indy.cred_def_id
+
+            root.payload.type = if $payload.exists("by_format.cred_offer") {
+                $payload.by_format.cred_offer.keys().index(0)
+            } else {
+                "indy"
+            }
+
+            # Optional:
+            root.payload.did = $payload.by_format.cred_offer.ld_proof.credential.issuer
+        }
+        if $category == "present_proof_v2_0" {
+            root.payload.verified = $payload.verified.or("false").bool()
+            root.payload.proof_id = "v2-" + $payload.pres_ex_id
+
+            # Optional fields (sometimes null):
+            root.payload.parent_thread_id = $payload.pres_request.id
+            root.payload.presentation = $payload.by_format.pres.indy
+            root.payload.presentation_request = $payload.by_format.pres_request.indy
+        }
+
+    # Step 2: Transform to Cloud API event
+    - label: transform_acapy_event_to_cloudapi_event
+      mapping: |
+        #!blobl
+        root = {
+            "topic": this.cloudapi_topic,
+            "wallet_id": this.metadata.get("x-wallet-id").or(this.metadata.origin).or("unknown"),
+            "origin": this.metadata.origin.or("unknown"),
+            "group_id": this.metadata.group_id.or("no_group"),
+            "payload": this.payload
+        }
+
+    - log:
+        level: TRACE
+        message: 'After step 2 for subject ${!@nats_subject}'
+        fields_mapping: |
+          #!blob
+          root.connect.message = this
+
+    # Step 3: Handle endorsement events and set NATS subjects
+    - label: handle_endorsement_events_and_set_nats_subjects
+      mapping: |
+        #!blobl
+        root = this
+
+        let is_endorsement_applicable = this.wallet_id == env("GOVERNANCE_ACAPY_LABEL").or("DIDx") &&
```

```suggestion
        let is_endorsement_applicable = this.wallet_id == env("GOVERNANCE_ACAPY_LABEL").or("Governance") &&
```
Looks like you copied from dev but in local its not "DIDx" but "Governance". Not sure why

---

Applying this on my local seems to resolve the test failures 🙌